sources/osg/Quat.js

```diff
@@ -101,17 +101,25 @@ define( [
 
 
         // http://physicsforgames.blogspot.fr/2010/02/quaternions.html
+        // called quatBlend
+        //  (  quaternion nlerp is supposed to be commutative, constant velocity, torque minimal )
```

Comment:
http://number-none.com/product/Understanding%20Slerp,%20Then%20Not%20Using%20It/ tells is not commutive